frontend/src/components/InnerNavbar/InnerNavbar.scss

```diff
@@ -0,0 +1,7 @@
+.SideBar {
+  width: 240px;
+  height: 100%;
+  padding: 20px;
+  border-top-left-radius: 20px;
+  background-color: rgba($color: #090d2c, $alpha: 0.5);
+}
```

Comment:
Sorry, forgot to mention this, but can we please add a newline at the end of the file